tests/test_yaml.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import pytest
+
+import helper
+from rpmlb.yaml import Yaml
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
```

Comment:
Because,
In case of that we implement like this.
If `valid_yaml` has error, I thought that it threw the error before `test_init_loads_file` was run. (I am not sure for that.)

```
def test_init_loads_file(valid_yaml):
    assert valid_yaml
```